src/ace_time/hw/StmRtc.h

```diff
@@ -29,7 +29,11 @@ class StmRtc {
   public:
     /** Constructor. */
     explicit StmRtc();
-
+    
+    /** Start RTC clock with clock source and format defined. Backwards compatible with defaults */
+    /** Select RTC clock source: LSI_CLOCK, LSE_CLOCK or HSE_CLOCK */
+    bool begin(const sourceClock_t clockSource = LSI_CLOCK, const hourFormat_t hourFormat = HOUR_FORMAT_24);
```

Comment:
Since `StmRtc` is a wrapper around `STM32RTC`, it seems like we should use the enums defined in `STM32RTC`, instead of the enums that come from the C source code `rtc.h`. So instead of `sourceClock_t` and `hourFormat_t`, it seems like it's better to make this:

```C++
bool begin(
    const STM32RTC::Source_Clock clockSource = STM32RTC::LSI_CLOCK, 
    const STM32RTC::Hour_Format hourFormat = STM32RTC::HOUR_24);
```

(Also, line wrapped to be < 80 columns.)